Wire-iOS Share Extension/UnsentSendable.swift

```diff
@@ -34,12 +34,17 @@ enum UnsentSendableError: Error {
     // The attachment is over file size limitation
     case fileSizeTooBig
 
+    // the target conversation does not exist anymore
+    case conversationNotExist
```

Comment:
Why not write out the whole sentence

```suggestion
    case conversationDoesNotExist
```